src/latexify/codegen/function_codegen_test.py

```diff
@@ -744,3 +744,284 @@ def test_use_set_symbols_compare(code: str, latex: str) -> None:
     tree = ast.parse(code).body[0].value
     assert isinstance(tree, ast.Compare)
     assert function_codegen.FunctionCodegen(use_set_symbols=True).visit(tree) == latex
+
+
+@test_utils.require_at_least(10)
+def test_matchvalue() -> None:
```

Comment:
Shouldn't these tests belong in `function_codegen_match_test.py`?

nit: most test methods in this file have the form `test_visit_XYZ_explanation`